database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -188,6 +188,13 @@ abstract class SlickDatabase(override val originalDatabaseConfig: Config) extend
     }
   }
 
+  protected[this] def isOfficiallySupportedDatabase(): Boolean = {
+    dataAccess.driver match {
+      case PostgresProfile | MySQLProfile | HsqldbProfile => true
```

Comment:
Presumably maria is included by `MySQLProfile`?

Reply:
Yes